### PR TITLE
Allow build without ProtonMail private configs

### PR DESCRIPTION
**Pull request by ksc91u**
This makes developers easier to build debug build.

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -36,7 +36,17 @@ kapt {
 }
 
 val privateProperties = Properties().apply {
-    load(FileInputStream("privateConfig/private.properties"))
+    try {
+        load(FileInputStream("privateConfig/private.properties"))
+    } catch (e: Exception) {
```
Comment (ghost) [inline app/build.gradle.kts:41]: I suggest catching only the specific exception this code intends to handle:
```suggestion
    } catch (e: FileNotFoundException) {
```

This way it won't mask other problems: for example the file exists but is incorrectly formatted and `load(...)` throws some parsing exception.

I also suggest logging the caught exception, again to be prudent.

Up to you and the maintainers/reviewers, I have no authority or knowledge of this project :)

Comment (fardavide) [reply app/build.gradle.kts:41]: Yes, limiting the caught exception is always a good choice. Logging is not mandatory here

Comment (fardavide) [reply app/build.gradle.kts:41]: This PR has been merged, as no updated has been received from the author and we decided that the requirements are satified

**Comment by fardavide** [comment]
This has been merged on main branch of GitLab repository.
Changes will be visible here once it reaches production.

Commit ref: `e2c4df4a`

**Pull request closed without merging**